actors/builtin/miner/miner_actor.go

```diff
@@ -76,12 +75,6 @@ type ConstructorParams = power.MinerConstructorParams
 func (a Actor) Constructor(rt Runtime, params *ConstructorParams) *adt.EmptyValue {
 	rt.ValidateImmediateCallerIs(builtin.InitActorAddr)
 
-	// Sanity check that we've been given a valid peer ID
-	_, err := peer.IDFromString(string(params.PeerId))
```

Comment:
Unless there is a reason to remove this a similar sanity check using `IDFromBytes` would still be helpful.

Reply:
Reasoning from @anorth here: https://github.com/filecoin-project/specs-actors/issues/425#issuecomment-638521129